### ISSUE TITLE
Update Syntax Visualizer VSIX for CTP 6

### ISSUE THREAD

**Issue by conniey** [dotnet/roslyn#1162]
I am trying to write an analyser with Rosyln with VS Ultimate CTP6, so I was following the README instructions.  But when I try to install
[.NET Compiler Platform Syntax Visualizer for CTP5](https://visualstudiogallery.msdn.microsoft.com/b5104545-29ed-46b2-beb0-351af9ca2d21), it fails with this log.

```
3/10/2015 12:33:56 AM - Microsoft VSIX Installer
3/10/2015 12:33:56 AM - -------------------------------------------
3/10/2015 12:33:56 AM - Initializing Install...
3/10/2015 12:33:57 AM - Extension Details...
3/10/2015 12:33:57 AM - 	Identifier      : 1e4ce699-d626-42c3-9803-cac3cd73ed77-CTP5
3/10/2015 12:33:57 AM - 	Name            : .NET Compiler Platform Syntax Visualizer for CTP5
3/10/2015 12:33:57 AM - 	Author          : Microsoft
3/10/2015 12:33:57 AM - 	Version         : 1.0.0.50123
3/10/2015 12:33:57 AM - 	Description     : The Syntax Visualizer is a Visual Studio extension that allows you to inspect and explore the syntax trees you'll use as you build applications and VS extensions atop the .NET Compiler Platform ("Roslyn").
3/10/2015 12:33:57 AM - 	Locale          : en-US
3/10/2015 12:33:57 AM - 	MoreInfoURL     : http://go.microsoft.com/fwlink/?LinkId=394587
3/10/2015 12:33:57 AM - 	InstalledByMSI  : False
3/10/2015 12:33:57 AM - 	SupportedFrameworkVersionRange : [4.5,)
3/10/2015 12:33:57 AM - 
3/10/2015 12:33:57 AM - 	SignedBy        : Microsoft Corporation
3/10/2015 12:33:57 AM - 	Certificate Info : [Subject]
  CN=Microsoft Corporation, OU=MOPR, OU=OPC, O=Microsoft Corporation, L=Redmond, S=Washington, C=US

[Issuer]
  CN=Microsoft Code Signing PCA, O=Microsoft Corporation, L=Redmond, S=Washington, C=US

[Serial Number]
  33000000CC776171D4B17BA1280001000000CC

[Not Before]
  4/22/2014 10:39:04 AM

[Not After]
  7/22/2015 10:39:04 AM

[Thumbprint]
  9DD77A79F7124D8DB128A526833CF3760FCB7794

3/10/2015 12:33:57 AM - 	Supported Products : 
3/10/2015 12:33:57 AM - 		Microsoft.VisualStudio.Pro
3/10/2015 12:33:57 AM - 			Version : [14.0.22512]
3/10/2015 12:33:57 AM - 
3/10/2015 12:33:57 AM - 	References      : 
3/10/2015 12:33:57 AM - 		-------------------------------------------------------
3/10/2015 12:33:57 AM - 		Identifier   : Microsoft.VisualStudio.MPF.14.0
3/10/2015 12:33:57 AM - 		Name         : Visual Studio MPF 14.0
3/10/2015 12:33:57 AM - 		Version      : [14.0]
3/10/2015 12:33:57 AM - 		MoreInfoURL  : 
3/10/2015 12:33:57 AM - 		Nested       : No
3/10/2015 12:33:57 AM - 
3/10/2015 12:33:57 AM - 
3/10/2015 12:33:57 AM - Searching for applicable products...
3/10/2015 12:33:57 AM - Found installed product - Microsoft Visual Studio 2010 Ultimate
3/10/2015 12:33:57 AM - Found installed product - Microsoft Visual Studio Ultimate 2015 CTP
3/10/2015 12:33:57 AM - Found installed product - Microsoft Visual Studio Premium 2015 CTP
3/10/2015 12:33:57 AM - Found installed product - Microsoft Visual Studio Professional 2015 CTP
3/10/2015 12:33:57 AM - Found installed product - Microsoft Visual Studio Community 2015 CTP
3/10/2015 12:33:57 AM - Found installed product - Microsoft Visual Studio 2015 CTP Shell (Integrated)
3/10/2015 12:33:57 AM - Found installed product - Global Location
3/10/2015 12:33:57 AM - VSIXInstaller.NoApplicableSKUsException: This extension is not installable on any currently installed products.
   at VSIXInstaller.App.InitializeInstall(Boolean isRepairSupported)
   at VSIXInstaller.App.InitializeInstall()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
```

**Comment by sharwell** [comment]
https://visualstudiogallery.msdn.microsoft.com/0f18f8c3-ec79-468a-968f-a1a0ee65b388

**Comment by Pilchie** [comment]
Tag @jmarolf

**Comment by jmarolf** [comment]
I'll update the readme with instructions for CTP6

**Comment by jmarolf** [comment]
fixed with pull request #1231

**Comment by gandarez** [comment]
tks for the web link!
